### PR TITLE
Feature/dhall csv tests

### PR DESCRIPTION
**Pull request by MarcosJLR**
I made some tests for the `dhall-csv` package and fixed some bugs that I discovered along the way (like dealing with negative integers in `csv-to-dhall` or Bools in `dhall-to-csv`).

I'm open to any suggestion on other things that I should test.

### CODE REVIEW

File: dhall-csv/src/Dhall/Csv.hs [hunk 1]
```diff
@@ -62,11 +62,16 @@ dhallToCsv e0 = listConvert $ Core.normalize e0
...
+    fieldConvert (Core.BoolLit True) = return $ toField ("true" :: Text)
+    fieldConvert (Core.BoolLit False) = return $ toField ("false" :: Text)
     fieldConvert (Core.NaturalLit a) = return $ toField a
     fieldConvert (Core.IntegerLit a) = return $ toField a
     fieldConvert (Core.DoubleLit (DhallDouble a)) = return $ toField a
     fieldConvert (Core.TextLit (Core.Chunks [] a)) = return $ toField a
     fieldConvert (Core.App (Core.Field (Core.Union _) _) a) = fieldConvert a
+    fieldConvert (Core.Field (Core.Union _) (Core.FieldSelection _ k _)) = return $ toField k
+    fieldConvert (Core.Some e) = fieldConvert e
+    fieldConvert (Core.App Core.None _) = return $ toField ("null" :: Text)
```
Comment (sjakobi) [inline dhall-csv/src/Dhall/Csv.hs:74]: My impression was that missing values are often represented as the empty string (`,,`). Is `null` widely used too?

**Review by sjakobi — APPROVED** [review]
Nice tests!

File: dhall-csv/src/Dhall/Csv.hs (hunk 1 above)
Comment (MarcosJLR) [reply dhall-csv/src/Dhall/Csv.hs:74]: I actually didn't give it much thought haha now that you say it is way better the empty string than null 😅 I'll change it right away

**Comment by MarcosJLR** [comment]
There are some problems with the `\r` character and Windows, I removed it from the tests because I don't have a Windows machine to test it and see what's happening. I will merge it like this but we should be aware to test that in the future.

**Pull request merged**